createdisk.sh

```diff
@@ -221,15 +236,15 @@ if ! ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo openssl x509 -checken
 fi
 
 # Add a user developer:developer with htpasswd identity provider and give it sudoer role
-${OC} --config $1/auth/kubeconfig create secret generic htpass-secret --from-literal=htpasswd=${DEVELOPER_USER_PASS} -n openshift-config
-${OC} --config $1/auth/kubeconfig apply -f htpasswd_cr.yaml
-${OC} --config $1/auth/kubeconfig create clusterrolebinding developer --clusterrole=sudoer --user=developer
+${OC} --config $clusterDirectory/auth/kubeconfig create secret generic htpass-secret --from-literal=htpasswd=${DEVELOPER_USER_PASS} -n openshift-config
+${OC} --config $clusterDirectory/auth/kubeconfig apply -f htpasswd_cr.yaml
+${OC} --config $clusterDirectory/auth/kubeconfig create clusterrolebinding developer --clusterrole=sudoer --user=developer
 
```

Comment:
I probably would have split the "clusterDirectory" change and the --help addition to a separate commit, it's then easier to see the "important" changes made by this commit (ie the tag-related ones).

Reply:
@cfergeau `$currentDirectory` comes from the tag related changes. since I have to implement the parse flag and this should be part of it. I am not sure if I split it out then the script even works as expected.

Reply:
You can start by adding usage() which will document -d / -h, and s/$1/$clusterDirectory, and then do the tag related changes on top of it. Or am I missing something which would prevent this from working?

Reply:
@cfergeau no, I can't do that since then I also have to remove otherwise any option we provide along with `cluster directory` this will not allow it to pass. And if I remove that then there `$1` is something is not an expected value so it is tie with current commit.

```bash
if [[ $# -ne 1 ]]; then
   echo "You need to provide the running cluster directory to copy kubeconfig"	
fi
```